src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -108,33 +108,34 @@ public void ILLink_runs_and_creates_linked_app(string targetFramework, bool refe
         }
 
         [RequiresMSBuildVersionTheory("16.8.0")]
-        [InlineData("netcoreapp3.0", "copyused")]
-        [InlineData("net5.0", "copyused")]
-        [InlineData("net5.0", "link")]
-        public void ILLink_links_simple_app_without_analysis_warnings_and_it_runs(string targetFramework, string trimMode)
+        [MemberData(nameof(SupportedTfms), MemberType = typeof(PublishTestUtils))]
```

Comment:
I'm curious how you decided whether to run a given test for the full matrix or just some configurations.

Reply:
My read of the test was that it should pass on all configurations -- not sure whether or not that's correct

Reply:
I think it should - my question was more about the criteria for testing all configs. This testcase in particular seems like a good idea as a smoke test, but a lot of them just test MSBuild logic which behaves the same for all TFMs and might not be worth it (this one for example https://github.com/dotnet/sdk/pull/18160/files#diff-f12d20c2aaef217fcb831a91991d26ddc5faac172673d49f4e59cb9db0015252R988).

Reply:
Yeah, that makes sense -- in that case we could consider making those `LatestTfm` only

Reply:
If you want to point out in the diff which ones you think are SDK-only, I can change those. Otherwise I'll err on the side of running all the frameworks.